com.unity.render-pipelines.core/Runtime/Common/DynamicResolutionHandler.cs

```diff
@@ -65,6 +65,31 @@ public class DynamicResolutionHandler
         private PerformDynamicRes m_DynamicResMethod = null;
         private static DynamicResolutionHandler s_Instance = new DynamicResolutionHandler();
 
+        /// <summary>
+        /// The scheduling mechanism to apply upscaling.
+        /// </summary>
+        public enum UpsamplerScheduleType
+        {
+            /// <summary>
```

Comment:
Just a fyi, if you want to make enum doc more compact you can use this syntax as well:
/// <summary>Description.</summary>
This way it's only 2 lines per value instead of 4:)